cypress/e2e/wallet-dropdown.test.ts

```diff
@@ -0,0 +1,34 @@
+import { getTestSelector } from '../utils'
+
+describe('Wallet Dropdown', () => {
+  before(() => {
+    cy.visit('/')
+  })
+
+  it.skip('should change the theme', () => {
```

Comment:
flag isn't selectable, so I'm going to just skip these until we have a branch where we can comment out.

Reply:
Why isn't it selectable?

Reply:
I'm guessing feature flags is bit conditionally on the build type and the build type probably disables feature flags.  I looked at the video that from the failing tests and the feature flag option wasn't there

Reply:
<img width="1035" alt="Screen Shot 2022-08-31 at 4 29 11 PM" src="https://user-images.githubusercontent.com/15934595/187777834-28f2be3e-6d9c-45cf-b81f-8dc0b5ad660d.png">


as you can see here, it times out b/c the flag doesn't exist.  I tested and ran this all locally.

Reply:
Either the env vars should be updated so that testing can cover this case, or the exported Context should be updated so that the flag hooks can be used in testing.
These shouldn't just be skipped - they're valuable tests.